hack/jenkins/e2e-runner.sh

```diff
@@ -192,6 +192,7 @@ function e2e_test() {
         echo "Publish version to ci/latest-green.txt: $(cat version)"
         gsutil cp ./version gs://kubernetes-release/ci/latest-green.txt
     fi
+    return ${exitcode}
```

Comment:
we need to be careful about this, I think - we need to make sure the cluster is torn down before we errexit.